<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="date">
     <date>
      <year>2022</year>
      <month>1</month>
      <day>1</day>
     </date>
    </property>
   </widget>
   <widget class="QCheckBox" name="pbMon">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>78</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Monday</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="pbWed">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Wednesday</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="pbTue">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>78</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Tuesday</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="pbThu">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>78</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Thursday</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="pbFri">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>78</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Friday</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="pbSat">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>78</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Saturday</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="pbSun">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>78</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Sunday</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>100</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>@Calculate@</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
